zerver/openapi/zulip.yaml

```diff
@@ -6630,6 +6630,13 @@ paths:
                 type: array
                 items:
                   type: object
+                  properties:
```

Comment:
I think it makes sense to include `additionalProperties` keyword, with a value of `false`, will indicate that only the two properties you define below are allowed. Generally in our documentation, this keyword comes before the properties are described.